app/portainer/views/settings/edge-compute/settingsEdgeCompute.html

```diff
@@ -0,0 +1,22 @@
+<rd-header>
```

Comment:
use react for the whole view?

Reply:
i'd rather leave this change for later, in case we have enough time, a new edge devices view still needs to be added, that can be done entirely in react and will require some more time